### DISCUSSION TITLE
LESSON 8 , when i click the connect button it just write connected without my metamask account popping up

### DISCUSSION THREAD

**Discussion by king-null** [Cyfrin/foundry-full-course-cu#1535] [Q&A]
i just cloned the html ,js code  from Patrick's github page and i was following the same procedure like him, but when he click the connect button metamask poped up, but when i did nothin showed ony to write connected. please do anyone else encounter this same issue?

this is the error i get in my chrome console

```
<== 1711526898662 sendError Error: Request of type 'PUBLIC_requestAccounts' already pending for origin http://127.0.0.1:5501. Please wait.
index.js:18 Error: Request of type 'PUBLIC_requestAccounts' already pending for origin http://127.0.0.1:5501. Please wait.
    at oa.<anonymous> (inpage.js:200:25462)
    at Generator.throw (<anonymous>)
    at ye (inpage.js:200:23826)
index.js:22 []
```

**Comment by EngrPips** [comment]
hello @king-null , What section is that on the Foundry-Full course, so that i can check it and maybe see what you might be doing wrong.

**Reply by king-null** [reply]
section 2 (introduction to window.ethereum)

**Comment by yawarasuuna** [comment]
I had the same problem while using Firefox, so I moved to a chromium based web browser, and it worked.

edit: I just noticed you mentioned you were already using chrome, sorry. I'll leave this up in case someone faces the same problem as I did

**Comment by EngrPips** [comment]
Interesting
